.github/workflows/release.yml

```diff
@@ -0,0 +1,317 @@
+name: Release new version of Optable SDK and demos
+on:
+  push:
+      tags:
+          - 'v.*'
+
+jobs:
+  tests-prettier:
+    uses: ./.github/workflows/reusable-lint-test.yml
+
+  build-web:
```

Comment:
QQ: should we use the `make -j --output-sync` used in circleCi to spread things up ?

Reply:
I didn't use the makefile for everything but I did use it to build the vanilla-demo, more information on the reasons were provided in the PR description.